client/my-sites/feature-upsell/store-upsell.jsx

```diff
@@ -110,6 +111,13 @@ class StoreUpsellComponent extends Component {
 							icon={ <img alt="" src="/calypso/images/illustrations/google-adwords.svg" /> }
 							title={ '$100 for Google AdWords' }
 							description={ 'Start bringing traffic immediately with Google AdWords.' }
+							body={
+								<div className="google-voucher__initial-step">
+									<TipInfo
+										info={ 'Offer valid in US after spending the first $25 on Google AdWords.' }
```

Comment:
is there a reason we aren't translating anything in this file?

Reply:
@dmsnell we want to see what it does to the sales funnel first, so before translating we are going to run an A/B test for english speaking audience only

Reply:
are we limiting it to accounts with English selected as their language?

Reply:
> so before translating we are going to run an A/B test for english speaking audience only

That's not unreasonable, but you do need to be careful.

It might be worth adding explanatory comments  where you've skipped `translate` calls, which would double as a string you can grep for if the experiment is succesful and does need to be translated.

Looking at the `TipInfo`, it sounds like you mean "US" rather than "English", or perhaps both.

That's a very small proportion, but in the context of an A/B test it's still a lot of noise.
Browser Language for 

> are we limiting it to accounts with English selected as their language?

We do still limit A/B tests to English by default: https://github.com/Automattic/wp-calypso/blob/6f6c7addeae62a79d82b36999cf43abc44d7ca72/client/lib/abtest/index.js#L105

However, `Offer valid in US` sounds like we're missing `countryCodeTargets` in `nudgeAPalooza`.